content/en/blog/_posts/2021-07-09-How-to-choose-a-SIG-as-a-non-code-Kubernetes-contributor.md

```diff
@@ -0,0 +1,48 @@
+
+My first work in the cloud native world was working with the [SIG Docs](https://github.com/kubernetes/community/tree/master/sig-docs) team. I helped review some documentation and learned some of their processes. I wasn't there long before I found out about another great SIG: [Contributor Experience](https://github.com/kubernetes/community/blob/master/sig-contributor-experience/README.md). Contributor Experience is all about improving the workflow, removing bottlenecks, and generally making Kubernetes a great project to contribute your time and effort. I knew ContribEx (as it’s called) is where I could make an impact.
+
+My philosophy is if things were hard or confusing for me, they'd likely be hard for a large swath of folks. Process improvements can come from anyone. Why not me?
+
+I found early on this: continually showing up to SIG meetings is one way to add value to the community. Regular participation gives you growing context in Kubernetes, and it helps in the long run.
+
+# How I contribute
+
+For me, contributing to Kubernetes is all about helping other contributors. I love talking to new contributors about why something is the way it is and they come up with some alternatives to doing that task. That thought exercise can be an enormously helpful contribution. That has been a part of what I've done in the community. Asking why something is the way it is can be a potent tool for improving things. I’ve helped with [new contributor workshops](https://github.com/kubernetes/community/blob/master/events/2018/05-contributor-summit/new-contributor-workshop.md), which meant showing up and sharing my knowledge. I’ve helped onboard folks into the Kubernetes GitHub organization. I’ve hosted community meetings.
+
+To be honest, I think the most complicated contributions to Kubernetes I’ve made involved the [community link shortener](https://github.com/kubernetes/k8s.io/blob/master/k8s.io/configmap-nginx.yaml). That consisted of editing a [ConfigMap](https://kubernetes.io/docs/concepts/configuration/configmap/). I had to ask for help once because something wasn’t apparent to me, I think, and there was feedback to create more consistency with already created URLs. But, at no point in time was anyone reluctant to help or the feedback negative. It was a genuinely fantastic experience. Plus, I don’t have to remember the YouTube page URL anymore. I type in [yt.k8s.io](https://yt.k8s.io). Speaking of, we have [YouTube, Slack, and other admins](/docs/comms/moderators/) for tools used throughout the community. They all need administration and, more importantly, participants. These things don’t necessarily require coding knowledge but familiarity with the tool that’s in use.
```

Comment:
```suggestion
To be honest, I think the most complicated contributions to Kubernetes I’ve made involved the [community link shortener](https://github.com/kubernetes/k8s.io/blob/master/k8s.io/configmap-nginx.yaml). That consisted of editing a [ConfigMap](https://kubernetes.io/docs/concepts/configuration/configmap/). I had to ask for help once because something wasn’t apparent to me, I think, and there was feedback to create more consistency with already created URLs. But, at no point in time was anyone reluctant to help or the feedback negative. It was a genuinely fantastic experience. Plus, I don’t have to remember the YouTube page URL anymore. I type in [yt.k8s.io](https://yt.k8s.io). Speaking of, we have [YouTube, Slack, and other admins](https://www.kubernetes.dev/docs/comms/moderators/) for tools used throughout the community. They all need administration and, more importantly, participants. These things don’t necessarily require coding knowledge but familiarity with the tool that’s in use.
```

Reply:
yes, thanks @jihoon-seo . Just noticed the links are not working. Will make the changes in a while.